### ISSUE TITLE
Click to add event from Month View

### ISSUE THREAD

**Issue by jamesdixon** [marzubov/ember-calendar#7]
Is it possible to create an event from the month view?

**Comment by marzubov** [comment]
@jamesdixon not yet, because initially events where with the time taken from clicked slot.
This functionality requires some popup like Google does,

**Comment by jamesdixon** [comment]
@marzubov I think this could be relative simple. Example:

``` hbs
{{as-calendar onDayClick=(action 'dayClicked')}}
```

```
// controller.js
actions: {
  dayClicked(event) {
    // do something with event (popup, etc)
    // `event` would contain the date of the day of month clicked
  }
}
```

Thinking that we leave most of the implementation up to the user -- all they would need is the date.

Thoughts?

**Comment by marzubov** [comment]
@jamesdixon Yes, I agree. We can just add action and leave event handling to user.

**Comment by jamesdixon** [comment]
@marzubov are you on the Ember Slack channel by chance?

**Comment by marzubov** [comment]
@jamesdixon Yes, just joined)

**Comment by jamesdixon** [comment]
@marzubov what's your username on Slack?

**Comment by marzubov** [comment]
@jamesdixon same, `marzubov`

**Comment by jamesdixon** [comment]
@marzubov I started working on this here: https://github.com/scoutforpets/ember-calendar/tree/add-day-click-event

An event can be added, but the positioning on the calendar is way off. Given your familiarity, I was hoping you might be able to adjust the calculation to work properly, please?
